@navikt/core/react/src/overlays/floating-menu/Menu.stories.tsx

```diff
@@ -0,0 +1,612 @@
+      </Menu.RadioGroup>
+    </MenuWithAnchor>
+  );
+};
+
+export const Animated = () => {
+  const files = ["README.md", "index.js", "page.css"];
+  const [file, setFile] = React.useState(files[1]);
+  const [open, setOpen] = React.useState(true);
+  const checkboxItems = [
+    { label: "Bold", state: React.useState(false) },
+    { label: "Italic", state: React.useState(true) },
+    { label: "Underline", state: React.useState(false) },
+    { label: "Strikethrough", state: React.useState(false), disabled: true },
+  ];
+
+  return (
+    <>
+      <label>
+        <input
+          type="checkbox"
+          checked={open}
+          onChange={(event) => setOpen(event.target.checked)}
+        />{" "}
+        open
+      </label>
+      <br />
+      <br />
+      <MenuWithAnchor className="contentAnimated content" open={open}>
+        {checkboxItems.map(
+          ({ label, state: [checked, setChecked], disabled }) => (
+            <Menu.CheckboxItem
+              key={label}
+              className="item"
+              checked={checked}
+              onCheckedChange={setChecked}
+              disabled={disabled}
+            >
+              {label}
+              <Menu.ItemIndicator>
+                <TickIcon />
+              </Menu.ItemIndicator>
+            </Menu.CheckboxItem>
+          ),
+        )}
+        <Menu.RadioGroup value={file} onValueChange={setFile}>
+          {files.map((_file) => (
+            <Menu.RadioItem key={_file} className="item" value={_file}>
+              {_file}
+              <Menu.ItemIndicator>
+                <TickIcon />
+              </Menu.ItemIndicator>
+            </Menu.RadioItem>
+          ))}
+        </Menu.RadioGroup>
+      </MenuWithAnchor>
+    </>
+  );
+};
+
+type MenuProps = Omit<
+  React.ComponentProps<typeof Menu> & React.ComponentProps<typeof Menu.Content>,
+  | "trapFocus"
+  | "onCloseAutoFocus"
+  | "disableOutsidePointerEvents"
+  | "disableOutsideScroll"
+>;
+
+const MenuWithAnchor: React.FC<MenuProps> = (props) => {
+  const { open = true, children, ...contentProps } = props;
+  return (
+    <Menu open={open} onOpenChange={() => {}} modal={false}>
+      {/* inline-block allows anchor to move when rtl changes on document */}
+      <Menu.Anchor>
+        <div style={{ display: "inline-block" }} />
+      </Menu.Anchor>
+      <Menu.Portal>
+        <Menu.Content
+          className="content"
+          onCloseAutoFocus={(event) => event.preventDefault()}
+          align="start"
+          {...contentProps}
+        >
+          {children}
+        </Menu.Content>
+      </Menu.Portal>
+    </Menu>
+  );
+};
+
+export const MenuWithOpenButton = () => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Menu open={open} onOpenChange={() => {}} modal={false}>
+      {/* inline-block allows anchor to move when rtl changes on document */}
+      <Menu.Anchor asChild>
+        <button
+          type="button"
+          onPointerDown={(event) => {
```

Comment:
https://ux.stackexchange.com/questions/130567/when-to-start-opening-the-select-list-on-mousedown-or-on-click-and-why

Can think of two potential solutions:
- Emulate native `<select>` better, where triggering pointerdown opens select, but if the pointer is moved away from the trigger or panel when released, it auto-closes. When clicked (onClick) it stays open.
- Emulate chrome bookmarks, where pointerdown only opens dropdown if you move the mouse over where the panel should open. 

I personally like 1 better since it emulates native behaviour, and the user wont have to learn a "hidden" pattern